cellfree/algorithm/get_end_motif.py

```diff
@@ -0,0 +1,118 @@
+import collections
+from dataclasses import dataclass
+
+import pandas as pd
+
+from cellfree.prepare.prepare_bam import read_prepared_bam_to_molecules
+
+# def get_k_mers(molecule):
```

Comment:
We have VCS (version control system) so we don't need to worry about removal of code.